package/yast2.spec

```diff
@@ -139,44 +117,44 @@ Provides:       y2t_menu
 Provides:       yast2-trans-menu
 Obsoletes:      y2t_menu
 Obsoletes:      yast2-trans-menu
-
 # moved here from another packages
-Provides:       yast2-dns-server:/usr/share/YaST2/modules/DnsServerAPI.pm
-Provides:       yast2-installation:/usr/share/YaST2/modules/Hotplug.ycp
-Provides:       yast2-installation:/usr/share/YaST2/modules/HwStatus.ycp
-Provides:       yast2-installation:/usr/share/YaST2/modules/Installation.ycp
-Provides:       yast2-installation:/usr/share/YaST2/modules/Product.ycp
+Provides:       yast2-dns-server:%{_datadir}/YaST2/modules/DnsServerAPI.pm
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/Hotplug.ycp
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/HwStatus.ycp
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/Installation.ycp
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/Product.ycp
 Provides:       yast2-mail-aliases
-Provides:       yast2-network:/usr/share/YaST2/modules/Internet.ycp
-Provides:       yast2-packager:/usr/lib/YaST2/servers_non_y2/ag_anyxml
-
-Requires:       yast2-ruby-bindings >= 3.1.33
-
-Summary:        YaST2 - Main Package
-License:        GPL-2.0
-Group:          System/YaST
+Provides:       yast2-network:%{_datadir}/YaST2/modules/Internet.ycp
+Provides:       yast2-packager:%{_prefix}/lib/YaST2/servers_non_y2/ag_anyxml
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+# for Punycode.rb (bnc#651893) - the idnconv tool is located in
```

Comment:
That would not work without touching the respective packages:

```
# cnf idnconv
                       
Program 'idnconv' is present in package 'bind-utils',
which is installed on your system.

Absolute path to 'idnconv' is '/usr/bin/idnconv'.
Please check your $PATH variable to see whether
it contains the mentioned path.

# rpm -q --provides bind-utils
bind-utils = 9.9.6P1-33.1
bind-utils(x86-64) = 9.9.6P1-33.1
bind9-utils
bindutil
config(bind-utils) = 9.9.6P1-33.1
dns_utils
```

Unfortunately package `bind-utils` does not explicitly provide `/usr/bin/idnconv`.